conf/application.conf

```diff
@@ -228,11 +228,11 @@ pagination {
   }
 
   departure {
-    numberOfMovements = 20
+    numberOfMovements = 10
   }
```

Comment:
We need to be careful with this as this is actually the live app config. It may be overriden in AppConfig base but we should probably not change this unless necessary